### PR TITLE
Change the file location to s3

### PR DESCRIPTION
**Pull request by sunank200**
# Description
## What is the current behavior?

HTTP URL in example DAG is failing. 

closes: #1296

## What is the new behavior?

- Change the HTTP URL for IRIS dataset to s3 URL in example_load_file

## Does this introduce a breaking change?
No

### Checklist
- [x] Created tests which fail without the change (if possible)
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

**Review by pankajastro — APPROVED** [review]

File: python-sdk/example_dags/example_load_file.py [hunk 1]
```diff
@@ -215,7 +214,7 @@
     # [END load_file_example_17]
 
     # [START load_file_example_18]
-    dataframe = aql.load_file(input_file=File(path=data_url, filetype=FileType.CSV))
+    dataframe = aql.load_file(input_file=File(path="s3://astro-sdk/sample_csv.data", filetype=FileType.CSV))
```
Comment (phanikumv) [inline python-sdk/example_dags/example_load_file.py:217]: what if a user wants to use this file ? It is not accessible unless thats a public s3 bucket.

Comment (phanikumv) [reply python-sdk/example_dags/example_load_file.py:217]: can we store it in `python_sdk/example_dags/data` folder?

Comment (sunank200) [reply python-sdk/example_dags/example_load_file.py:217]: But the case we are handling as part of this example is that the file is of a different extension than `csv` but the filetype is `csv`.

**Review by phanikumv — APPROVED** [review]

File: python-sdk/example_dags/example_load_file.py (hunk 1 above)
Comment (sunank200) [reply python-sdk/example_dags/example_load_file.py]: Changed the folder location as suggested here: https://github.com/astronomer/astro-sdk/pull/1297/commits/91eab7b161577a51bcc0445c17cdcf9a2d96f6ab

Comment (sunank200) [reply python-sdk/example_dags/example_load_file.py]: I can create a separate issue for the following

> I am more concerned about the fact that we are making the data not available to users who want to try out this DAG for the first time

Comment (sunank200) [reply python-sdk/example_dags/example_load_file.py]: Created a issue for this https://github.com/astronomer/astro-sdk/issues/1299
@phanikumv

File: python-sdk/example_dags/example_load_file.py [hunk 2]
```diff
@@ -34,7 +33,7 @@
 with dag:
     # [START load_file_example_1]
     my_homes_table = aql.load_file(
-        input_file=File(path="s3://astro-sdk/sample.csv"),
+        input_file=File(path="s3://astro-sdk/python_sdk/example_dags/data/sample.csv"),
```
Comment (pankajkoti) [inline python-sdk/example_dags/example_load_file.py:36]: Perhaps we can `s3://astro-sdk/python_sdk/example_dags/data` as a prefix constant

Comment (sunank200) [reply python-sdk/example_dags/example_load_file.py:36]: Created a issue for this https://github.com/astronomer/astro-sdk/issues/1299

**Pull request merged**